packages/website/src/docs/standard-library/openapi.md

```diff
@@ -0,0 +1,234 @@
+---
+id: openapi
+title: The OpenAPI v3 emitter
+---
+
+<!-- cspell:ignore cadl, openapi -->
```

Comment:
Is it something we should configure enforcement for? If not not sure we should add tool specific suppression that doesn't apply to everybody

Reply:
It would be nice if we enforced doc spell checking and had a shared dictionary file for things like this. Perhaps we can track that as a separate infra issue?

I don't think a few comments like this in the meantime are an issue. We could keep a note on the tracking issue for enforcement to go clean them up.

Reply:
fair enough, will create an issue

Reply:
I added that line because I have cspell enabled locally. I am fine either way on making it enforced.